WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -0,0 +1,255 @@
+import CoreData
```

Comment:
Maybe better to do in a separate PR, but would it be possible to abstract away anything that is `CoreData` related?

Reply:
Well, regarding the abstraction everything can be discussed.  But this header isn't really needed so I went ahead and removed it.

Reply:
Marking as resolved for now but feel free to respond if there's anything else.